conf/app.routes

```diff
@@ -59,6 +59,10 @@ GET        /manage-your-tax-agents/relationship-not-found
 # Agent Journey URLs ----------------------------------------------------------------------------------------------------------
 GET        /:journeyType                                                                uk.gov.hmrc.agentclientrelationshipsfrontend.controllers.journey.StartJourneyController.startJourney(journeyType: AgentJourneyType)
 
+# TODO: THESE ARE TEMP NAMES
+GET        /:journeyType/fast-track-landing                                             uk.gov.hmrc.agentclientrelationshipsfrontend.controllers.journey.FastTrackLandingController.show(journeyType: AgentJourneyType)
```

Comment:
Probably best to name it something like need-more-information to match the content on the page